packages/collector/test/tracing/protocols/apollo_subgraph/client.js

```diff
@@ -10,15 +9,15 @@ require('../../../..')();
 const bodyParser = require('body-parser');
 const express = require('express');
 const morgan = require('morgan');
-const rp = require('request-promise');
+const fetch = require('node-fetch');
```

Comment:
request-promise is also a deprecated package, so I changed to node-fetch

Reply:
See ticket 150327.

We either do the library replacements in a separate pull request
or you prepare two mergeable commits in this pull request - as you wish :)

Reply:
I had noticed this on the deprecated list, but during the logic refactoring for Apollo Federation, I opted to remove it from the code. We can handle the deprecation of other usages in a separate pull request.